internal/postgres/pg_utils.go

```diff
@@ -105,3 +106,16 @@ func extractDatabase(url string) (string, error) {
 	}
 	return pgCfg.Database, nil
 }
+
+func registerTypesToConnMap(ctx context.Context, conn *pgx.Conn) error {
+	var hstoreExists bool
+	err := conn.QueryRow(ctx, "SELECT EXISTS (SELECT 1 FROM pg_type WHERE typname = 'hstore')").Scan(&hstoreExists)
```

Comment:
Would it not be simpler to just have one query to get the oid of the hstore type, and register the type if rows are found rather than performing two queries? If this is going to be done on every conn, we should try to make it as fast as possible 🏎️ .